app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RecipeIngredientsController < ApplicationController
+  def create
```

Comment:
This is an excellent controller design choice for creating an association between an ingredient and a recipe. Using `create` in a joins controller is definitely the way to go to best adhere to REST!

Nice work including error handling here! It's not required for an IC but is great practice!